source/_posts/2020-11-23-tplink-local-access.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: "TP-Link offers way to add local API back"
+description: "TP-Link last week decided to publish a firmware upgrade that removed access to the local API. They are partially reverting this decision."
+date: 2020-11-23 00:01:00
+date_formatted: "November 23, 2020"
+author: Paulus Schoutsen
+author_twitter: balloob
+categories: Public-Service-Announcement
+og_image: /images/blog/2020-11-23-tplink/forum-post-send-ticket.png
+---
+
+Last week TP-Link released an update for their HS100 and HS110 plugs that removed the local API. This was done because of a "security concern". I put this in quotes because it has not been verified and this reason has been given before when removing interopability. TP-Link communicated this via Twitter in response to a user voicing their concern.
```

Comment:
```suggestion
Last week TP-Link released an update for their HS100 and HS110 plugs that removed the local API. This was done because of a "security concern". I put this in quotes because it has not been verified and this reason has been given before when removing interoperability. TP-Link communicated this via Twitter in response to a user voicing their concern.
```